src/gt4py/next/ffront/decorator.py

```diff
@@ -240,86 +262,89 @@ def _implicit_offset_provider(self) -> common.OffsetProvider:
                         )
         return implicit_offset_provider
 
+    def _init_compiled_programs(self) -> None:
```

Comment:
Shouldn't this be the body of a `_compiled_programs` cachedproperty?

Reply:
I don't understand the comment.